### ISSUE TITLE
[UI Dashboard Time Machine] Switch to script editor and add function starts with cursor in odd place

### ISSUE THREAD

**Issue by karel-rehor** [influxdata/influxdb#16854]
__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

Generate some test data

1. Create a dashboard
2. Create a cell and define the initial query with query builder.  Save the cell. 
3. Click configure on the cell again.  Time machine opens in Query builder view. 
4. Click on the Script Editor.  The query appears as a flux script. 
```javascript
from(bucket: "qa")
  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
  |> filter(fn: (r) => r._measurement == "beat")
  |> filter(fn: (r) => r._field == "pulse")
```
5. Click on add a function.  e.g. aggregateWindow 

__Expected behavior:__
Describe what you expected to happen.

Expected the function to be added at a logical place in the pipeline. For example at the end of the last call to a filter function generated from query builder. 

__Actual behavior:__
Describe What actually happened.

Function is added at the _start_ of the pipeline.

```javascript
from(bucket: "qa")
  |> aggregateWindow(every: 1m, fn: mean)
  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
  |> filter(fn: (r) => r._measurement == "beat")
  |> filter(fn: (r) => r._field == "pulse")
```
Submitting this results in the following error message. 
```
error in building plan while starting program: cannot submit unbounded read to "qa"; try bounding 'from' with a call to 'range'
```
**_Generally_**

Would expect pipeline functions to be appended after the last function in the pipeline.  

This can cause confusion for beginner or casual users. 

__Screencast__

![FluxEditInitialCursorPlacement01](https://user-images.githubusercontent.com/25881301/74441744-a7fe8380-4e70-11ea-8ec8-d3defab55c0c.gif)

__Environment info:__

* System info: Run `uname -srm` and copy the output here
```
Linux 4.15.0-76-generic x86_64
```
* InfluxDB version: Run `influxd version` and copy the output here
Detected in nightly build. 
```
ts=2020-02-13T11:50:50.800628Z lvl=info msg="Welcome to InfluxDB" log_id=0Kwb2Ch0000 version=nightly commit=a17471696 build_date=2020-02-13T11:50:50Z
```
* Other relevant environment details: Container runtime, disk info, etc
Browser: __Chrome 79__

__Config:__
Copy any non-default config values here or attach the full config as a gist or file.

Started with e2e testing enabled. 

```
--e2e-testing=true
```

**Comment by drdelambre** [comment]
possibly related to #16668

**Comment by desa** [comment]
Will be fixed with #16668
